src/content/docs/en/pages/devtools/cli/azion-cli/azion-overview/azion-overview.mdx

```diff
@@ -39,7 +39,7 @@ The Azion CLI is built in Go and interacts with Azion products through the Azion
 If you choose the **RPM**, **Dpkg**, or **apk** package manager, or the **.deb** file, go to the [releases page](https://github.com/aziontech/azion-cli/releases) and download the desired package.
 
 :::note 
-If you choose to use Homebrew, the previous step isn't necessary.
+If you choose to use Homebrew, Chocolatey, or Windows Package Manager, the previous step isn't necessary.
```

Comment:
After Windows Package Manager, you can put in parenthesis (Winget). A lot of people know this manager by it's shorthand.